<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestBoardControlWidget</class>
 <widget class="QWidget" name="TestBoardControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <widget class="QGroupBox" name="statusGroup">
       <property name="title">
        <string>设备状态</string>
       </property>
       <layout class="QFormLayout" name="statusLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>状态:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="statusEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>序列号:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="serialNumberEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>主电源(313):</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="mainPowerStatusLabel">
          <property name="text">
           <string>未知</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="mainControlGroup">
       <property name="title">
        <string>主控制</string>
       </property>
       <layout class="QGridLayout" name="mainControlLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>开始采集</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="stopButton">
          <property name="text">
           <string>停止采集</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="powerOnButton">
          <property name="text">
           <string>DUT上电</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="powerOffButton">
          <property name="text">
           <string>DUT断电</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="queryVoltageButton">
          <property name="text">
           <string>查询电压</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="queryFaultButton">
          <property name="text">
           <string>查询故障</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="dutGroup">
     <property name="title">
      <string>DUT选择</string>
     </property>
     <layout class="QVBoxLayout" name="dutLayout">
      <item>
       <layout class="QHBoxLayout" name="topSelectionLayout">
        <item>
         <widget class="QCheckBox" name="selectAllCheckBox">
          <property name="text">
           <string>全选</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="mainPowerCheckBox">
          <property name="text">
           <string>主电源 (313)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mainPowerStatusIndicator">
          <property name="styleSheet">
           <string notr="true">color: red; font-weight: bold;</string>
          </property>
          <property name="text">
           <string>OFF</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="dutsGridLayout"/>
      </item>
      <item>
       <widget class="QLabel" name="helpLabel">
        <property name="styleSheet">
         <string notr="true">color: gray;</string>
        </property>
        <property name="text">
         <string>提示: 请在勾选需要监控的DUT后，再点击&quot;开始采集&quot;。不勾选发送断电才视为断电</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="toolBarLayout">
     <item>
      <widget class="QPushButton" name="parseBinFileButton">
       <property name="text">
        <string>解析Bin日志文件</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="advancedGroup">
     <property name="title">
      <string>高级/调试指令</string>
     </property>
     <layout class="QGridLayout" name="advancedLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>芯片类型:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="chipTypeComboBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="setChipTypeButton">
        <property name="text">
         <string>设置芯片类型</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>标定指令(Hex):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="calibCmdEdit">
        <property name="placeholderText">
         <string>例如: 0102030A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="calibSendButton">
        <property name="text">
         <string>发送标定指令</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>写寄存器:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="regWriteAddrEdit">
        <property name="placeholderText">
         <string>地址(Hex)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="regWriteDataEdit">
        <property name="placeholderText">
         <string>数据(Hex)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="regWriteSendButton">
        <property name="text">
         <string>写入</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>读寄存器:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="regReadAddrEdit">
        <property name="placeholderText">
         <string>地址(Hex)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="regReadLengthEdit">
        <property name="placeholderText">
         <string>长度(Dec)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="regReadSendButton">
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="dataTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>DUT</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>芯片类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gyro X (LSB)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gyro Y (LSB)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gyro Z (LSB)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Acc X (LSB)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Acc Y (LSB)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Acc Z (LSB)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mix (LSB)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Temp (°C)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomLayout">
     <item>
      <layout class="QVBoxLayout" name="logControlLayout"/>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: red; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
